src/get_clades_to_model.py

```diff
@@ -21,14 +21,17 @@
 # /// script
 # requires-python = ">=3.11"
 # dependencies = [
-#   "virus_clade_utils@git+https://github.com/reichlab/virus-clade-utils/",
+#   "virus_clade_utils@git+https://github.com/reichlab/virus-clade-utils",
```

Comment:
(This had been pointing to a feature branch for testing, but said feature has been merged, so switched it back...without the unsightly `/`)